src/user/user.controller.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Controller,
+  Get,
+  Param,
+  UseGuards,
+  ParseUUIDPipe,
+} from '@nestjs/common';
+import { UserService } from './user.service';
+import { AuthGuard } from 'src/auth/guards/auth.guard';
+import { User } from '@prisma/client';
+import { CurrentUser } from './decorators/current.user.decorator';
+
+@Controller('')
```

Comment:
You can add `AuthGuard` on `Controller` level.

Then you can remove it on endpoint level.

```suggestion

@UseGuards(AuthGuard)
@Controller('')
```